database/src/main/scala/cromwell/database/migration/metadata/table/FailureEventTableMigration.scala

```diff
@@ -0,0 +1,62 @@
+package cromwell.database.migration.metadata.table
+
+import cromwell.database.migration.metadata.MetadataCustomSql
+
+class FailureEventTableMigration extends MetadataCustomSql {
+  import MetadataCustomSql._
+
+  override def queries: Array[String] = {
+    Array(
+      """
+        |INSERT INTO METADATA_JOURNAL (
+        |  WORKFLOW_EXECUTION_UUID,
+        |  METADATA_KEY,
+        |  CALL_FQN,
+        |  JOB_SCATTER_INDEX,
+        |  JOB_RETRY_ATTEMPT,
+        |  METADATA_VALUE,
+        |  METADATA_VALUE_TYPE,
+        |  METADATA_TIMESTAMP
+        |)
+        |SELECT
+        |  WORKFLOW_EXECUTION_UUID,
+        |  CONCAT('failures[', fe.FAILURE_EVENT_ID ,']:failure'),
```

Comment:
same question as below about using a synthetic PK as a subscript for this metadata